### PR TITLE
docs: remove entity module references

### LINKED ISSUE #1075: missing doc page
https://googlecloudplatform.github.io/gcloud-node/#/docs/v0.27.0/datastore/entity

### PR DESCRIPTION
**Pull request by callmehiphop**
closes #1075

### CODE REVIEW

File: lib/datastore/request.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ function DatastoreRequest() {}
  * @param {Key|Key[]} keys - Datastore key object(s).
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request
- * @param {module:datastore/entity|module:datastore/entity[]} callback.entity -
+ * @param {object|object[]} callback.entity -
```
Comment (stephenplusplus) [inline lib/datastore/request.js]: We can probably change the description, maybe "The entity object(s) which match the provided keys."

Comment (callmehiphop) [inline lib/datastore/request.js]: done!

File: lib/datastore/request.js [hunk 2]
```diff
@@ -559,7 +559,7 @@ DatastoreRequest.prototype.delete = function(keys, callback) {
  *     query with this object to check for more results.
  * @param {object} callback.apiResponse - The full API response.
  * @param {?error} callback.err - An error returned while making this request
- * @param {module:datastore/entity[]} callback.entities - A list of Entities
+ * @param {object[]} callback.entities - A list of Entities
```
Comment (stephenplusplus) [inline lib/datastore/request.js]: Description here too!

Comment (callmehiphop) [inline lib/datastore/request.js]: Whoops!

**Pull request merged**